### PR TITLE
feat: cluster-client support for mm.app

### PR DESCRIPTION
**Pull request by shaoshuai0102**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -17,6 +18,7 @@ const MOCK_APP_METHOD = [
   '_readyCallbacks',
   'closed',
   'close',
+  'agent',
```
Comment (shaoshuai0102) [inline lib/app.js]: 上个 mr 把这个漏掉了，导致 app.agent 取不到

**Comment by popomore** [comment]
我发个版本

**Comment by popomore** [comment]
+ egg-mock@2.3.0

**Comment by popomore** [comment]
https://github.com/eggjs/egg/issues/170

**Pull request merged**